### PR TITLE
[cape2.sh] minor updates

### PR DESCRIPTION
**Pull request by zoomequipd**

### CODE REVIEW

File: Sandbox/cape2.sh [hunk 1]
```diff
@@ -532,16 +532,16 @@ function install_suricata() {
...
+    : ` 
+    You can now start suricata by running as root something like "/usr/bin/suricata -c /etc/suricata//suricata.yaml -i eth0".
 
     If a library like libhtp.so is not found, you can run suricata with:
     LD_LIBRARY_PATH=/usr/lib /usr/bin/suricata -c /etc/suricata//suricata.yaml -i eth0
 
     While rules are installed now, its highly recommended to use a rule manager for maintaining rules.
     The two most common are Oinkmaster and Pulledpork. For a guide see:
     https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Rule_Management_with_Oinkmaster
-    """
```
Comment (zoomequipd) [inline Sandbox/cape2.sh:544]: should this just be removed?

Comment (doomedraven) [reply Sandbox/cape2.sh:544]: yes just remove it please, thanks

**Comment by doomedraven** [comment]
ready to merge or wait?

**Comment by zoomequipd** [comment]
should be good.

**Comment by doomedraven** [comment]
thanks

**Pull request merged**